roles/kubernetes/preinstall/tasks/main.yml

```diff
@@ -60,7 +60,6 @@
     - "{{ kube_config_dir }}/ssl"
     - "{{ kube_manifest_dir }}"
     - "{{ kube_script_dir }}"
-    - "{{ local_volume_provisioner_base_dir }}"
 
```

Comment:
@rsmitty the creation of base_dir already included in original 2.0 upgrade, see https://github.com/kubernetes-incubator/kubespray/blob/64020040180556c9b835dfb18b54cbd5b5ba0a17/roles/kubernetes-apps/external_provisioner/local_volume_provisioner/tasks/main.yml#L3

That’s why I just remove the redundant creation of base_dir from here :-)